environment.yml

```diff
@@ -0,0 +1,184 @@
+    - greenlet==3.0.3
+    - grpcio==1.60.0
+    - h11==0.14.0
+    - h5py==3.10.0
+    - huggingface-hub==0.20.3
+    - idna==3.6
+    - imageio==2.33.1
+    - importlib-metadata==7.0.1
+    - importlib-resources==6.1.1
+    - jaxtyping==0.2.19
+    - jinja2==3.1.3
+    - kiwisolver==1.4.5
+    - langcodes==3.3.0
+    - lazy-loader==0.3
+    - linear-operator==0.5.2
+    - mako==1.3.1
+    - markdown==3.5.2
+    - markupsafe==2.1.4
+    - matplotlib==3.7.4
+    - medmnist==3.0.1
+    - monotonic==1.6
+    - mpmath==1.3.0
+    - murmurhash==1.0.10
+    - netcal==1.3.5
+    - networkx==3.1
+    - numexpr==2.8.6
+    - nvidia-cublas-cu12==12.1.3.1
+    - nvidia-cuda-cupti-cu12==12.1.105
+    - nvidia-cuda-nvrtc-cu12==12.1.105
+    - nvidia-cuda-runtime-cu12==12.1.105
+    - nvidia-cudnn-cu12==8.9.2.26
+    - nvidia-cufft-cu12==11.0.2.54
+    - nvidia-curand-cu12==10.3.2.106
+    - nvidia-cusolver-cu12==11.4.5.107
+    - nvidia-cusparse-cu12==12.1.0.106
+    - nvidia-ml-py3==7.352.0
+    - nvidia-nccl-cu12==2.18.1
+    - nvidia-nvjitlink-cu12==12.3.101
+    - nvidia-nvtx-cu12==12.1.105
+    - oauthlib==3.2.2
+    - opt-einsum==3.3.0
+    - packaging==23.2
+    - pandas==2.0.3
+    - pillow==10.2.0
+    - preshed==3.0.9
+    - progress==1.6
+    - protobuf==4.25.2
+    - psutil==5.9.8
+    - py3nvml==0.2.7
+    - pyasn1==0.5.1
+    - pyasn1-modules==0.3.0
+    - pycparser==2.21
+    - pydantic==2.5.3
+    - pydantic-core==2.14.6
+    - pyparsing==3.1.1
+    - pyro-api==0.1.2
+    - pyro-ppl==1.9.0
+    - python-dateutil==2.8.2
+    - pytz==2023.3.post1
+    - pywavelets==1.4.1
+    - pyyaml==6.0.1
+    - regex==2023.12.25
+    - requests==2.31.0
+    - requests-oauthlib==1.3.1
+    - restrictedpython==7.0
+    - rsa==4.9
+    - ruamel-yaml==0.18.5
+    - ruamel-yaml-clib==0.2.8
+    - safetensors==0.4.2
+    - scikit-image==0.21.0
+    - segment-analytics-python==2.2.3
+    - sentry-sdk==1.39.2
+    - setproctitle==1.3.3
+    - six==1.16.0
+    - smart-open==6.4.0
+    - smmap==5.0.1
+    - sniffio==1.3.0
+    - soundfile==0.12.1
+    - soupsieve==2.5
+    - spacy==3.7.2
+    - spacy-legacy==3.0.12
+    - spacy-loggers==1.0.5
+    - sqlalchemy==1.4.51
+    - srsly==2.4.8
+    - starlette==0.35.1
+    - sympy==1.12
+    - tensorboard==2.14.0
+    - tensorboard-data-server==0.7.2
+    - tensorboardx==2.6.2.2
+    - termcolor==2.4.0
+    - thinc==8.2.2
+    - tifffile==2023.7.10
+    - tikzplotlib==0.9.8
+    - timm==0.9.12
+    - tokenizers==0.15.1
+    - torch==2.1.2
+    - torchaudio==2.1.2
+    - torcheval==0.0.7
+    - torchvision==0.16.2
+    - tqdm==4.66.1
+    - transformers==4.37.0
+    - triton==2.1.0
+    - typeguard==2.13.3
+    - typer==0.9.0
+    - typing-extensions==4.9.0
+    - tzdata==2023.4
+    - urllib3==2.1.0
+    - uvicorn==0.27.0
+    - wandb==0.16.2
+    - wasabi==1.1.2
+    - weasel==0.3.4
+    - werkzeug==3.0.1
+    - wfdb==4.1.2
+    - xmltodict==0.13.0
+    - zipp==3.17.0
+prefix: /home/aditya/anaconda3/envs/usb
```

Comment:
Can we make this generic?

Reply:
I did not get this point.

Reply:
prefix: /home/aditya/anaconda3/envs/usb can we make this generic?